src/Distribution/Extra/Doctest.hs

```diff
@@ -100,6 +100,16 @@ import Distribution.PackageDescription
        (CondTree (..))
 #endif
 
+#if MIN_VERSION_Cabal(3,0,0)
+import Distribution.Simple.Utils (findFileEx)
+#else
+import Distribution.Simple.Utils (findFile)
+#endif
+
+#if MIN_VERSION_Cabal(3,0,0)
```

Comment:
Perhaps combine this import with the earlier use of `MIN_VERSION_Cabal(3,0,0)` to minimize CPP?

Reply:
I feel uneasy here, I don't like merging CPP blocks which have multiple needs **and** `#else` branches.
i.e. it might look like I need `libraryNameString` for `findFile` functionality.

That said, I'm not consistent.